assets/js/components/adminbar/AdminBarImpressions.js

```diff
@@ -111,4 +115,16 @@ AdminBarImpressions.defaultProps = {
 	className: 'mdc-layout-grid__cell--span-2-tablet mdc-layout-grid__cell--span-3-desktop',
 };
 
+/**
+ * Has Zero Data
+ *
+ * Allows parent component to check if this component has zero data.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Function} select Data store select function.
+ * @return {Function} Select function for parent component to run through useSelect hook.
+ */
+AdminBarImpressions.hasZeroData = ( select ) => reportArgs === undefined ? undefined : isZeroReport( select( MODULES_SEARCH_CONSOLE ).getReport( reportArgs ) );
```

Comment:
Let's call these `selectHasZeroData` to make it a bit more explicit that these functions require `select` to be provided.